lighthouse-core/gather/gatherers/dobetterweb/tags-blocking-first-paint.js

```diff
@@ -43,7 +43,7 @@ function installMediaListener() {
       // @ts-expect-error - `___linkMediaChanges` created above.
       window.___linkMediaChanges.push(mediaChange);
 
-      return this.setAttribute('media', val);
+      this.setAttribute('media', val);
```

Comment:
I think the intention here was to be double sure we were matching the behavior of the replaced property setter...but eslint correctly points out that even if you return a value in a setter the value still goes nowhere, so the `return` is misleading